### DISCUSSION TITLE
Kata revision history page & 500 Internal Server Error

### DISCUSSION THREAD

**Discussion by Yang-Wei-Ting** [codewars/content-issues#160] [Q&A]
Hi I'm the author of this kata https://www.codewars.com/kata/620bce448e757800242821fe

Recently, nine out of ten tries I cannot access the kata revision history page due to 500 Internal Server Error.

Perhaps I revised the kata too many times?
If so, may I squash some earlier commits?

---
Edit

Just noticed that maybe this thread should be moved to 
https://github.com/codewars/codewars.com/discussions instead.

My apology if I put it in the wrong place.

**Comment by Blind4Basics** [comment]
"closing", as requested. But neither places were bad, imo. ;)

**Comment by hobovsky** [comment]
Created a dedicated issue: https://github.com/codewars/codewars.com/issues/2932
